TFT/src/User/API/parseACK.c

```diff
@@ -682,7 +682,7 @@ void parseACK(void)
                infoFile.source >= BOARD_MEDIA && infoFile.source <= BOARD_MEDIA_REMOTE &&
                ack_seen("SD printing byte"))  // if printing from (remote) onboard media
       {
-        setPrintResume(false);
+        setPrintResume(true);
```

Comment:
that should remain setPrintResume(false). setPrintResume(true) is called in hostActionCommands function. The opposite happens for setPrintPause() function (true in parseAck function and false in false in hostActionCommands function). Basically, for resume we wait the **last** confirmation from hostActionCommands function. For pause, we wait the **first** confirmation from parseAck function.

Reply:
The true-false argument for setPrintResume is redundant.